backend/routes/users.js

```diff
@@ -122,9 +122,8 @@ router.get('/users/profile', function (req, res, next) {
 /*
 POST /users/profile
 */
-router.post('/users/profile', function (req, res, next) {
-    // Get id of user inside of JWT
-    let id = 1; // TODO
+router.post('/users/profile', authentication.isNotAnonymous, function (req, res, next) {
```

Comment:
it's a put request defined in the subject
```suggestion
router.put('/users/profile', authentication.isNotAnonymous, function (req, res, next) {
```

Reply:
Ok, it's done